src/actions/index.js

```diff
@@ -0,0 +1,74 @@
+import { 
+  SET_AUTH_USER, 
+  REMOVE_AUTH_USER, 
+  ADD_ITEM_TO_CART, 
+  REMOVE_ITEM_FROM_CART, 
+  CONFIRM_ORDER, 
+  CANCEL_ORDER,
+  ADD_ITEM_TO_WISHLIST,
+  REMOVE_ITEM_FROM_WISHLIST,
+  UPDATE_WISHLIST_ITEM } from '../constants/actionTypes';
+
+// User actions
+
+export const setAuthUser = (authUser) => ({
+  type: SET_AUTH_USER,
+  authUser,
+  isUserLoggedIn: authUser ? true : false
```

Comment:
@vijaykarthikpro @Rasika-Hatwar It's a bad structure. you call `setAuthUser` only when the user is logged in. `isUserLoggedIn` should always be true.